act-rules-format.bs

```diff
@@ -170,41 +192,89 @@ When all expectations are true for a test target, the test target `passed` the r
+
+<p class="note" role="note">
+"Editor's note: We are considering merging this definition with test definition. We are looking for feedback."
 </p>
 
-A [test target](#output-test-target) will pass a rule group if it passes at least one of the applicable rules within that group. Likewise, a test target will fail the rule group, if it does not pass any of the applicable rules in the group. In the example of HTML Tables, passing the table `scope` rule, and failing the ARIA label and `headers` + `id` rule, would mean the test target passed this group of rules.
+Aggregation Applicability {#aggregation-applicability}
+------------------------------------------
+
+ The applicability of a [composed rule](#composed-rule-outline) is defined as the union of all the applicability sections of its [atomic rules](#atomic-rule-outline). Because of this, applicability of a composed rule can be inferred from the atomic rules it aggregates. Since the applicability can be inferred, rule authors MAY omit applicability from the aggregation definition. This can be useful if it is difficult to express the combined applicability in plain language.
+
+<div class=example>
+  <p>A composed rule about `img` elements aggregates results from atomic rules that have the following applicability:</p>
+  <ul>
+    <li><strong>Atomic rule1</strong>: All `img` element with an `alt` attribute</li>
+    <li><strong>Atomic rule1</strong>: All `img` element without an `alt` attribute</li>
+  </ul>
+
+  <p>The applicability of the composed rule combines the applicability of both atomic rules. This becomes:</p>
+  <blockquote>
+    <p>All `img` elements.</p>
+  </blockquote>
+</div>
+
+
+Aggregation Expectations {#aggregation-expectations}
+----------------------------------------
+
+A [composed rule](#composed-rule-outline) MUST contain one or more expectations. An expectation is an assertion, written in plain language, that must be true about the outcomes of [atomic rules](#atomic-rule-outline) listed in [aggregated rules](#aggregation-definition). A composed rule expectation MUST NOT use information from [test aspects](#input-aspects) or from the [test applicability](#test-applicability).
+
+When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations is false, the test target `failed` the rule. This works the same way for atomic rules.
+
+<div class="example">
+  <p>A composed rule for [WCAG 2.1 Audio Description or Media Alternative](https://www.w3.org/TR/WCAG21/#audio-description-or-media-alternative-prerecorded) aggregates three atomic rules. The expectation of the composed rule is as follows:</p>
+  <blockquote>
+    <p>For each test target, the outcome of one of the following rules is `passed`:</p>
+    <ul>
+      <li>Video elements have an audio description</li>
+      <li>Video elements have a media alternative</li>
+      <li>Video elements is media alternative</li>
+    </ul>
+  </blockquote>
+</div>
+
 
```

Comment:
The list has mixes of singular and plural elements.  "Video elements" (plural) with "a", "an" or "is" (singular).